### PR TITLE
Restore coach filters

### LINKED ISSUE #9886: Update messages for filters on CoachExamsPage (and lessons too)
I spotted this while reviewing #9863 and saw some linting changes in the diff for `CoachExamsPage` where there are comments indicating that we were temporarily commenting out the Active/Inactive filters for exams in #6174 due to not having updated the strings.

With the guidance from @radinamatic in #6174 we need to be sure we have the correct strings defined in the component and that the select dropdown is visible and working as expected.

Note that `CoachExamsPage` isn't the only place where this is an issue, there are others as can be seen in https://github.com/learningequality/kolibri/pull/6233/files (the PR that commented the filters out).

### PR DESCRIPTION
**Pull request by akolson**
## Summary

This PR restores coach filters disabled in https://github.com/learningequality/kolibri/issues/6174. Also, based on feedback from @jtamiace an additional filter option **Ended** has been added to the quizes filter and appropriate filtering logic added.

Screenshot 2022-12-21 at 19 06 36
Screenshot 2022-12-21 at 19 07 06
Screenshot 2022-12-21 at 19 08 24
Screenshot 2022-12-21 at 19 09 33

## References

Closed #9886 

## Reviewer guidance

- Use the lessons and quiz filters under Coach Reports and Plan tabs as shown in the screenshots attached

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by nucleogenesis — APPROVED** [review]
Code LGTM! I'd wait for @radinamatic 's review on the strings before merging.

**Comment by pcenov** [comment]
@akolson and @radinamatic LGTM both filters are reinstated and are functioning correctly as specified above. Note that the strings are not localized though.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: I'm confused, why are these strings re-activated? There is no concept of **Active/Inactive** any more, neither for lessons nor for quizzes... 😕 

Quizzes can be **Not started**, **Started** and **Ended**.
Lessons can be **Visible** and **Not visible** (to learners).

We should delete all the instances of _Active/Inactive_ strings when they are supposed to refer to lessons or quizzes.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Not sure what exactly is this condition supposed to do, but filtering by _active exams_ makes no sense any more...

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Same as above for _active exams_.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: There should be no filters for active/inactive lessons or quizzes.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue
Comment (akolson) [reply kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: @radinamatic  These are specific messages displayed if no exams are returned on filtering. The confusion arises(apologies on the confusion!) with the names of the translation strings. These have been refactors to reflect started, not started and ended.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue
Comment (akolson) [reply kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: @radinamatic `printLabel` string seems to be used for file names during exports(Ln 184) + also as a report header(Ln 10)

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue
Comment (akolson) [reply kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Apologies on the confusion, the tranlation strings have been refactored to visible and not visible. This are messages displayed incase the filter returns no rows.

**Comment by akolson** [comment]
@radinamatic translation string changes have been effected and pr is ready for a re-review. Thanks

**Review by marcellamaki — APPROVED** [review]
Final updates look good, and manual re-review after last adjustments check out.

**Pull request merged**
